tracker/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+
+
+# Create your models here.
+class User(models.Model):
+    name = models.CharField(max_length=256)
+    last_Name = models.CharField(max_length=256)
+    login = models.CharField(max_length=256, unique=True)
+    password = models.CharField(max_length=256)
+
+
+class Customer(User):
```

Comment:
I suggest leaving one User class instead of User, Customer and Admin without separation. In the User class we could have a field called “status” with two options – “CUSTOMER” and “ADMIN”. Why do we need a User table, a Customer table and an Admin table?
Also, there is a user admin in the system. So it is justified